TLC5955.cpp

```diff
@@ -191,7 +191,7 @@ void TLC5955::setControlModeBit(bool is_control_mode)
   SPI.begin();
 }
 
-void TLC5955::updateLeds()
+int TLC5955::updateLeds()
 {
   // Get number of counts for current pattern
   uint32_t power_output_counts = 0;
```

Comment:
This whole computation is not necessary.

Reply:
fixed